sklearn/utils/fixes.py

```diff
@@ -332,3 +332,14 @@ def _object_dtype_isnan(X):
     from collections import Iterable as _Iterable  # noqa
     from collections import Mapping as _Mapping  # noqa
     from collections import Sized as _Sized  # noqa
+
+
+# TODO: replace by copy=False, when only scipy > 1.1 is supported.
+def _astype_copy_false(X):
+    """Returns the copy=False parameter for {ndarray,csr_matrix,csc_matrix}.astype
```

Comment:
PEP8

Reply:
Thanks for the review! You are right, it's too long (although technically it passed flake8) -- Fixed.